CONTRIBUTING.md

```diff
@@ -29,9 +29,15 @@ run the tests, it can be useful to prepare a
 [virtual environment](https://docs.python.org/3/tutorial/venv.html) and install
 certain libraries typeshed uses to check stub files.
 
-Follow platform-specific instructions below. Following that, to automatically
-check your code before committing, you can copy the file `pre-commit` to
-`.git/hooks/pre-commit`.
+Follow platform-specific instructions below.
+
+The repository is equipped with a [`pre-commit`](https://pre-commit.com/)
+configuration file. If you want some checks to run automatically when you
+commit, you can install the hooks:
```

Comment:
Why would anyone want this? It would only run black and isort, and those run automatically when you make a pull request anyway.

Reply:
While I understand this might not be everyone's choice, I appreciate that some people like to post clean commits. Also on a more pragmatic point of view, this avoids launching about 40 jobs to cancel them after a few seconds, which has an (albeit small, yet) existing environmental impact :)

But I understand your point, in that we can make it clearer that this step is 100% optional.

Reply:
Is it good for you now ?

Reply:
Thanks! I like it now :)